src/style/components/WeeklyComponent.scss

```diff
@@ -0,0 +1,33 @@
+@import "../base/variables.scss";
+
+
+.WeeklyComponent {
+  background-color: #D9D9D9;
+  max-width: 350px;
+  padding: 20px;
+  border-radius: 10px;
+
+  h2 {
+    font-weight: 400;
+    margin-bottom: 10px;
+  }
+}
+
+.WeeklyComponentList {
+  list-style: none;
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+}
+
+.WeeklyComponentItem {
+  display: flex;
+  justify-content: space-between;
+  // gap: 10px;
+  padding-bottom: 10px;
+  border-bottom: 1px solid rgb(188, 188, 188);
+}
+
+.WeeklyComponentDay {
+  width: 90px;
+}
```

Comment:
I think the file name should be "WeeklyComponent.module.scss" to use scss in the WeeklyComponent.tsx file.